internal/namespaces/instance/v1/testdata/test-create-server-simple-default.stdout.golden

```diff
@@ -33,3 +34,4 @@ security-group.id         49822f6c-1d32-465c-a017-720691d159b2
 security-group.name       Default security group
 state-detail              
 arch                      x86_64
+zone
```

Comment:
why is there no zone in create, but a zone is set in get ?

Reply:
The issue comes from the cassette because the field zone was not returned by the API until recently.